libs/commons/JSON.ml

```diff
@@ -15,6 +15,14 @@ type t =
   | Null
 [@@deriving show]
 
+let member m j =
```

Comment:
Should probably be called lookup at this point, 

Also maybe time to define a JSON.mli to better document the API of this module.

Reply:
The equivalent function in Yojson is named [`member`](https://ocaml-community.github.io/yojson/yojson/Yojson/Basic/Util/index.html#val-member) (albeit that one raises an exception, rather than returning an option), so I was going for consistency here. Also, I would find member more descriptive than lookup in this context.

Happy to make a .mli file.

Reply:
@aryx in that case, do you understand the "(* compatibility mode with json-wheel *)" comment? Not sure where that should end up/relevancy thereof (can't find adding commit---looks like git history got lost since you moved this all from pfff).